src/goto-checker/report_util.cpp

```diff
@@ -71,6 +71,62 @@ void report_failure(ui_message_handlert &ui_message_handler)
   }
 }
 
+void report_inconclusive(ui_message_handlert &ui_message_handler)
+{
+  messaget msg(ui_message_handler);
+  msg.result() << "INCONCLUSIVE" << messaget::eom;
```

Comment:
Don't we already use "UNKNOWN" in places (goto-analyzer, for example)?

Reply:
I don't mind. Any preferences?